### ISSUE TITLE
Feature request: Auto add component export to barrel index file.

### ISSUE THREAD

**Issue by chiangs** [ricardoemerson/create-react-tsx-component#14]
Request option to automatically add the component to a barrel index when generated inside a directory.

Example is if the component is generated in a `shared` directory and the shared directory has a barrel `index.tsx` that exports  components from.
```
// shared/index.ts
export { default as ComponentName } from './ComponentName'
```

`import { ComponentName } from './shared'`

**Comment by santosfrancisco** [comment]
Hi @chiangs, I open that PR to include this options. https://github.com/ricardoemerson/create-react-tsx-component/pull/18 Do you think it works for you too?

**Comment by ricardoemerson** [comment]
Hello my friends. Sorry for being gone. Soon I will be evaluating your pull-request.

**Comment by ricardoemerson** [comment]
Feature included in version 1.7.0, but only for named components.
